cmd/influxd/run/server.go

```diff
@@ -116,6 +118,13 @@ func (s *Server) appendUDPService(c udp.Config) {
 	}
 	srv := udp.NewService(c)
 	srv.Server.PointsWriter = s.PointsWriter
+}
+
+func (s *Server) appendContinuousQueryService(c continuous_querier.Config) {
```

Comment:
Can we disable cq's in the config like we could in the old system?  If so, this might be a good spot to check and not launch them like we do with other services.